{
  "dec80529c845d3c27a85c4ffa4a3e36b" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "89bbff1923754f58",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384479,
        "stop" : 1757985384479,
        "duration" : 0
      }
    } ]
  },
  "8da926fc454e1bd488b226989b7a2548" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "865fb0b19d552582",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385568,
        "stop" : 1757985385568,
        "duration" : 0
      }
    } ]
  },
  "c1b6ae4b58275aff3d4dbb3084a65a12" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "196fa8f95910930a",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385009,
        "stop" : 1757985385009,
        "duration" : 0
      }
    } ]
  },
  "95e988f7b9ae1ffc1bb1a94df27947ab" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "49330d319437b8c9",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385469,
        "stop" : 1757985385469,
        "duration" : 0
      }
    } ]
  },
  "3603c4c02749c89693b5133729d25ccf" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "3c0ec721a4e8d6f9",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385556,
        "stop" : 1757985385556,
        "duration" : 0
      }
    } ]
  },
  "34ec45e163e33663f384eafd6389ba11" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "7226c541b8786d9",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385512,
        "stop" : 1757985385512,
        "duration" : 0
      }
    } ]
  },
  "74e11d64ebca920bc538b40564cd7c54" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "a3a24a9b27012407",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384452,
        "stop" : 1757985384452,
        "duration" : 0
      }
    } ]
  },
  "d2619fdb1ce04b6355472ae9d177e7a5" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "8b5a1eb6f33f3425",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985344305,
        "stop" : 1757985344305,
        "duration" : 0
      }
    } ]
  },
  "44766886e95a65e8fe0b09f4171df3fd" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "76c64250d1671ab2",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385494,
        "stop" : 1757985385494,
        "duration" : 0
      }
    } ]
  },
  "4538358d6042ac465d0fe40e25045202" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "b094c36cdca2d25e",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385529,
        "stop" : 1757985385529,
        "duration" : 0
      }
    } ]
  },
  "441638748dad4bb3f3a3d2b8c4d3eade" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "b709611b352445cf",
      "status" : "passed",
      "time" : {
        "start" : 1757985344254,
        "stop" : 1757985344302,
        "duration" : 48
      }
    } ]
  },
  "62528b176734532ae1579c983a37a7f7" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 1,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "41555babf7143665",
      "status" : "skipped",
      "statusDetails" : "Skipped: 跳过此用例",
      "time" : {
        "start" : 1757985384478,
        "stop" : 1757985384478,
        "duration" : 0
      }
    } ]
  },
  "4adc3905df29e6a824c2a2ecf542462e" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "7f7183a5206728af",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385546,
        "stop" : 1757985385546,
        "duration" : 0
      }
    } ]
  },
  "45fb348d02d7dad44caf0fc42cfac185" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "70d1857a8f2aeedc",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385519,
        "stop" : 1757985385519,
        "duration" : 0
      }
    } ]
  },
  "183f2c77153af6fada891c5bf3c281f7" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "584ed27c595835d5",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384491,
        "stop" : 1757985384491,
        "duration" : 0
      }
    } ]
  },
  "eedeea8d90538367d1fbcf318d491102" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "606c1fe01893cac7",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384504,
        "stop" : 1757985384504,
        "duration" : 0
      }
    } ]
  },
  "c1ae0a1dd077ca2ff24bf7b28b0c3ebc" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "4b73313eb1fe3552",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385484,
        "stop" : 1757985385484,
        "duration" : 0
      }
    } ]
  },
  "5ca588a38c2173ce305282401ff4c130" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "ced276ad8b5704ad",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985385580,
        "stop" : 1757985385580,
        "duration" : 0
      }
    } ]
  },
  "5e9761927d5b4ebfcf03f3c69136c9f8" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "cdffdecb5980e578",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384465,
        "stop" : 1757985384465,
        "duration" : 0
      }
    } ]
  },
  "f0473c39e8a96d1aa883e46395165381" : {
    "statistic" : {
      "failed" : 1,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 0,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "58c1c52f433ebd1c",
      "status" : "failed",
      "statusDetails" : "Failed: uat环境登录失败（已重试5次）: HTTPSConnectionPool(host='dev.lgcopytrade.top', port=443): Max retries exceeded with url: /api/sys/randomImage/1757985384409?_t=1757985384 (Caused by ProxyError('Unable to connect to proxy', FileNotFoundError(2, 'No such file or directory')))",
      "time" : {
        "start" : 1757985384440,
        "stop" : 1757985384440,
        "duration" : 0
      }
    } ]
  },
  "e5a6e08f8ee8c1aff161dc168228f371" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "9901a32a6d5b3b30",
      "status" : "passed",
      "time" : {
        "start" : 1757985384950,
        "stop" : 1757985385006,
        "duration" : 56
      }
    } ]
  },
  "54ce3513c9d82a843de1c3b796842a3f" : {
    "statistic" : {
      "failed" : 0,
      "broken" : 0,
      "skipped" : 0,
      "passed" : 1,
      "unknown" : 0,
      "total" : 1
    },
    "items" : [ {
      "uid" : "1f6c03c33af8cde9",
      "status" : "passed",
      "time" : {
        "start" : 1757985385419,
        "stop" : 1757985385469,
        "duration" : 50
      }
    } ]
  }
}